apps/condo/domains/marketplace/components/MarketSetting/MarketSettingForm.tsx

```diff
@@ -0,0 +1,147 @@
+import { MarketSetting as MarketSettingType } from '@app/condo/schema'
+import { Col, Form, Row } from 'antd'
+import { Gutter } from 'antd/es/grid/row'
+import get from 'lodash/get'
+import { useRouter } from 'next/router'
+import React, { useMemo } from 'react'
+
+import { useIntl } from '@open-condo/next/intl'
+import { ActionBar, Button } from '@open-condo/ui'
+import { Checkbox } from '@open-condo/ui'
+
+import { FormWithAction } from '@condo/domains/common/components/containers/FormList'
+import { useNotificationMessages } from '@condo/domains/common/hooks/useNotificationMessages'
+import { INVOICE_PAYMENT_TYPE_CASH, INVOICE_PAYMENT_TYPE_ONLINE, INVOICE_PAYMENT_TYPES } from '@condo/domains/marketplace/constants'
+import { MarketSetting } from '@condo/domains/marketplace/utils/clientSchema'
+
+
+const CHECKBOX_LAYOUT_PROPS = {
+    labelCol: {
+        span: 24,
+        md: 5,
+    },
+    wrapperCol: {
+        span: 24,
+        md: 6,
+    },
+    styled: {
+        paddingBottom: '12px',
+    },
+    colon: false,
+}
+const BIG_ROW_GUTTERS: [Gutter, Gutter] = [0, 60]
+const MIDDLE_ROW_GUTTERS: [Gutter, Gutter] = [0, 40]
+const SMALL_ROW_GUTTERS: [Gutter, Gutter] = [0, 20]
+
+interface IMarketSettingForm {
+    marketSetting?: MarketSettingType,
+    userOrganizationId: string
+    loading: boolean
+}
+export const MarketSettingForm: React.FC<IMarketSettingForm> = ({ marketSetting, userOrganizationId, loading }) => {
+    const intl = useIntl()
+    const allowCashPaymentTypeLabel = intl.formatMessage({ id: 'pages.condo.settings.marketplace.cashCheckbox.label' })
+    const allowOnlinePaymentsTypeLabel = intl.formatMessage({ id: 'pages.condo.settings.marketplace.onlineCheckbox.label' })
+    const SaveMessage = intl.formatMessage({ id: 'Save' })
+
+    const { getSuccessfulChangeNotification } = useNotificationMessages()
+
+    const router = useRouter()
+
+    const updateHook = MarketSetting.useUpdate({}, () => router.push('/settings?tab=marketplace'))
+    const updateAction = async (data) => {
+        await updateHook(data, marketSetting)
+    }
+    const createAction = MarketSetting.useCreate({ organization: { connect: { id: userOrganizationId } } }, () => router.push('/settings?tab=marketplace'))
+
+    const action = marketSetting ? updateAction : createAction
+
+    const initialValues = useMemo(() => {
+        const result = {}
+        for (const type of INVOICE_PAYMENT_TYPES) {
+            result[type] = false
+        }
+        const residentAllowedPaymentTypes = get(marketSetting, 'residentAllowedPaymentTypes')
+        for (const index in residentAllowedPaymentTypes) {
+            result[residentAllowedPaymentTypes[index]] = true
+        }
+        return result
+    }, [marketSetting])
```

Comment:
Did I understand correctly that initially all organizations do not have this setting, and if there is no such setting, then all types of payments are allowed?

If so, then it is not clear why when opening the form I will see checkboxes with the value of false